### DISCUSSION TITLE
体彩绽放——体育高光导引系统

### DISCUSSION THREAD

**Discussion by KyleHuang9** [InternLM/Tutorial#431] [project]
项目名称与介绍：体彩绽放，是一个体育高光导引系统（Sports Highlight Guiding System），以其卓越的智能定位和生动的解说能力，如同一场绚丽的体育花火，将每一个赛事高潮点点燃，为观众带来视听的双重盛宴。

项目定位：搭建体育赛事高光导引系统，以期能自动定位体育高光时刻并自动生成解说文本。
项目组成员：
        [KyleHuang9](https://github.com/KyleHuang9)
        [HaroldChen19](https://github.com/HaroldChen19)
        [water2silver](https://github.com/water2silver)

**Comment by zhangzhidaSunny** [comment]
想问下视频解析和高光时刻定位是用的哪个模型和技术呢？ 不知能门槛高不高，否一起做。
